### PR TITLE
docs(operators): add documentation for operator from

### LINKED ISSUE #77: Add docs for operator : creation - from
Add docs for operator from under group creation 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by mustafamg**
closes #77

### CODE REVIEW

File: src/operator-docs/creation/from.ts [hunk 1]
```diff
@@ -1,6 +1,91 @@
...
+      code: `
+      const array = [10, 20, 30];
+      const result = Rx.Observable.from(array);
+      result.subscribe(x => console.log(x));
+
+      // Results in the following:
+      // 10 20 30`,
+      externalLink: {
+        platform: 'JSBin',
+        url: 'http://jsbin.com/qodocay/edit?js,console'
```
Comment (ashwin-sureshkumar) [inline src/operator-docs/creation/from.ts]: please make the link as `embed` instead of `edit`

Comment (mustafamg) [reply src/operator-docs/creation/from.ts]: Okay.

**Comment by mustafamg** [comment]
Done

**Comment by ladyleet** [comment]
thx @mustafamg !

**Pull request merged**